.editorconfig

```diff
@@ -91,10 +91,10 @@ dotnet_code_quality_unused_parameters = all:suggestion
 
 #### C# Coding Conventions ####
 
-# var preferences
-csharp_style_var_elsewhere = false:silent
-csharp_style_var_for_built_in_types = false:silent
-csharp_style_var_when_type_is_apparent = true:silent
+# Avoid var
```

Comment:
😕 Why?

Reply:
Because it makes it harder to reason about types. Especially so when the code is reviewed outside VS.